frontend/src/static/js/pages/styles/PlaylistPage.scss

```
@@ -479,23 +505,34 @@
 				font-size: 13px;
 				font-weight: 500;
 				border: 0;
+				color: var(--text-muted);
 				background: none;
 				cursor: pointer;
 				padding: 0;
+
+				.circle-icon-button {
+					color: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)